src/libraries/Fuzzing/DotnetFuzzing/Dictionaries/typename.dict

```diff
@@ -0,0 +1,26 @@
+"	"
+"0"
+","
+"[*["
+" System.Int32  "
+"System.Int32&&"
+"System.Int32[,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,]"
+"System.Int32[*******************************]"
+"System.Int32[][][][][][][][][][][][][][][][][][][][]"
+".NoNamespace"
+# "Namespace.Kość" 
```

Comment:
Commented this line because of `ParseDictionaryFile` error, non-ASCII character, how to make it work @MihaZupan?
```log
ParseDictionaryFile: error in line 11
                "Namespace.Ko┼¢─ç"
```

Reply:
Ah never mind, apparently need to use the code not the characters

Reply:
I don't think this file will provide the fuzzer with the inputs you're expecting.

These lines are treated as literals - lists of bytes.
Since you're using `MemoryMarshal.Cast<byte, char>(bytes)` in your fuzzer, interpreting these bytes as UTF16 literals.

Consider
```c#
new string(MemoryMarshal.Cast<byte, char>("System.Int32[****]"u8));
```
, which is `"祓瑳浥䤮瑮㈳⩛⨪崪"`.

---

If you want to use bytes that aren't printable ASCII, you can write them in this file as hex `\xNN`.

Reply:
Yeah, tried with `\u015B\u0107` did not work, error at runtime
```log
ParseDictionaryFile: error in line 11
                "Namespace.Ko\u015B\u0107"
```

Reply:
`\x15B\x107` works, thanks!

Reply:
```suggestion
"Namespace.Ko\x15B\x107" 
```